### DISCUSSION TITLE
How to appear invisible in the room?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#397] [Q&A] [answered]
Hello, Sorry for the title. I do not know how to word this question I have. I would like to add a way in networked aframe to connect to the scene and see everyone but not have my avatar head there. What. I mean is I can see everyone but they can not see me. Is there a way I can do this?

Joseph

**Comment by vincentfretin** [comment, marked as answer]
Hello, you should really hang out on slack, other persons other than me can answer those type of questions, it's actually a general question here not really specific to naf.
You can achieve this by adding a `visible` property on your `player-info` component and in `update()` set your avatar mesh accordingly, for example `yourEl.object3D.visible = this.data.visible`. Add a button to hide your avatar like this `rig.setAttribute('player-info', 'visible:false')`

**Comment by Therealjosephchrzempiec** [comment]
I never been on slack. I don't have it download or an account.

I only just found out about aframe on stack overflow yesterday.

**Comment by vincentfretin** [comment]
You don't need to install the Slack app, I don't have it, I just use a pinned tab in my browser.
You can also join Discord. See the community page at https://aframe.io/community/

**Comment by vincentfretin** [comment]
FYI I enabled Discussions on this repo, please create a discussion if it's not a bug or feature request from now on.
